plugins/env/app/models/environment_variable.rb

```diff
@@ -56,7 +56,9 @@ def resolve_secrets(project, deploy_group, env, preview:)
           found = resolver.read(secret_key)
           resolved =
             if preview
-              found ? "#{value} ✓" : "#{value} X"
+              path = resolver.expand_key(secret_key)
+              expanded_path = !path.nil? ? path : ""
```

Comment:
this would be simpler ?
`path = resolver.expand_key(secret_key) || ""`

Reply:
also `secret://` seems like an odd response ...

Reply:
maybe something like
```Ruby
path = resolver.expand_key(secret_key)
value = path ? "secret://#{path} " : "#{value} X"
```

Reply:
i'd say a response with arbitrarily picked characters is a worse response than something that tries to resolve the path, and it's nowhere to be found.

A consumer of said response needs to know the prefix `secret://`, and then pick up the path to actually look in vault by taking the rest of that string, which will be empty.

For this model, you'd need special logic where "if the string ends with ` X`, we know there's no secret to be resolved at the other end.

I do know what i prefer, and what i care about is data models/structures -- and not the actual code that munges them, but in this case i kinda need to.